pkgs/by-name/sc/scom/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finaAttrs: {
```
Lets use `finalAttrs` to make it simpler to override (especially with some of the quirks with `src`)
- #315337